README.md

```diff
@@ -68,10 +68,13 @@
   * [URL Shortener](https://play.google.com/store/apps/details?id=de.keineantwort.android.urlshortener)
 - iOS
   * [ShortFox](https://itunes.apple.com/us/app/shortfox/id1412452706?ls=1&mt=8)
+  * [Short Menu](https://shortmenu.com/) - YOURLS support requires a one-time in-app purchase of about $2 USD (See also: [Setup instructions](https://shortmenu.com/support/custom-services/yourls/)).
```

Comment:
I would link to the iOS page specifically, since one exists:
```suggestion
  * [Short Menu](https://shortmenu.com/ios/) - YOURLS support requires a one-time in-app purchase of about $2 USD (See also: [Setup instructions](https://shortmenu.com/support/custom-services/yourls/)).
```